lib/couchx/prepare_query.ex

```diff
@@ -17,7 +17,7 @@ defmodule Couchx.PrepareQuery do
 
   @operator_keys Keyword.keys(@operators)
 
-  def call(%{wheres: wheres, limit: _limit} = query) do
+  def call(%{wheres: wheres, limit: _limit, offset: _offset} = query) do
```

Comment:
is offset always present?

Reply:
As far as I understand it is always present even if it is nil because it is a part of #Ecto.Query received, but seeing it this is not necessary, removed